final_prep/annotations.rb

```diff
@@ -4,40 +4,65 @@
 
 # Build a Bear
 
+# Creates a function called build_a_bear that takes five parameters
 def build_a_bear(name, age, fur, clothes, special_power)
+  # Designates argument greeting and returns a string with name interpolated in the result.
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # Passes argument demographics to return an array including our name and age arguments.
   demographics = [name, age]
+  # Passes argument power_saying and returns a string with special_power argument interpolated.
   power_saying = "Did you know that I can #{special_power}?"
+  # Creates a hash named build_bear stores data about built_bear.
```

Comment:
This is called "declaring a variable", not passing an argument, for `greeting`, `demographics`, and `power_saying`